cli/flox/src/commands/activate.rs

```diff
@@ -245,6 +235,16 @@ impl Activate {
             ("FLOX_PROMPT_COLOR_2", prompt_color_2),
         ]);
 
+        // Set `FLOX_PROMPT_ENVIRONMENTS` to either the constructed prompt string
+        // or "" if the prompt is disabled.
+        //
+        // The latter is necessary for inner actiavtions to override the policy
```

Comment:
nit: not quite clear to me what "the policy" means
```suggestion
        // The latter is necessary for inner activations to override the policy
```

Reply:
changed the description